webapp/graphite/dashboard/views.py

```diff
@@ -14,7 +15,7 @@
 from graphite.compat import HttpResponse
 from graphite.dashboard.models import Dashboard, Template
 from graphite.render.views import renderView
-from send_graph import send_graph_email
+from .send_graph import send_graph_email
```

Comment:
The future import isn't needed to make this work - explicit relative imports work in Py2 as well.

The future import makes it a bit harder to add back implicit relative imports (i.e. what I changed this from). I can remove it if you prefer.

Reply:
I think it makes sense to just use the absolute import and not bother with the __future__ import since the tests will all fail for py3.x if someone tries to use an implicit relative import